code/datums/quick_load_outfits.dm

```diff
@@ -0,0 +1,1138 @@
+/datum/outfit/quick
+	///Description of the loadout
+	var/desc = "Description here"
+	///How much of this loadout there is. infinite by default
+	var/quantity = -1
+	///What job this loadout is associated with. Used for tabs and access.
+	var/jobtype = "Squad Marine"
+
+
+/datum/outfit/quick/equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	pre_equip(H, visualsOnly)
```

Comment:
It's different only because it qdel's their shit (so you don't lose important gear because you picked up an item/have a round start bag on or some shit).
That said, I just discovered delete_equipment. Can I just go 
delete_equipment
. = ..()

Or does that proc do naughty things that make it unsuitable?